src/components/ui/MonacoEditor.svelte

```diff
@@ -89,7 +89,7 @@
       automaticLayout,
       fixedOverflowWidgets,
       language,
-      lightbulb: { enabled: true },
+      // lightbulb: { enabled: true },
```

Comment:
should this just be removed too?

Reply:
Believe so, should be enabled by default now i think.